tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
```

Comment:
Is this really necessary? We are not using tox in other Conan repos, I am fine if Python 3.6 is hardcoded in CI, and I'd prefer to keep it simpler

Reply:
not really needed, only in case need to validate multiple versions. Fine, python 3.6 is good enough, let me revert it

Reply:
Done, tox has been removed.